change/@ni-nimble-components-8e3914f8-cdb3-4a05-a43c-f5062fa29d85.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
According to [this comment in the linked issue](https://github.com/ni/nimble/issues/935#issuecomment-1370110263) there's a scrolling workaround that can be removed from the Blazor example app now too.

Reply:
Yep...working on that now 👍